suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml

```diff
@@ -258,10 +292,14 @@ tests:
               - "radosgw-admin user create --uid=repuser --display_name='Replication user' --access-key 3way123 --secret 3way123 --rgw-realm india --system"
               - "radosgw-admin zone modify --rgw-realm india --rgw-zonegroup shared --rgw-zone primary --access-key 3way123 --secret 3way123"
               - "radosgw-admin period update --rgw-realm india --commit"
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_realm india"
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_zonegroup shared"
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_zone primary"
-              - "ceph orch restart {service_name:shared.pri}"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_zone primary"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_zone primary"
```

Comment:
@mkasturi18 , could you disable sync for io rgw's in the next PR as an enhancement

Reply:
ack @hmaheswa , added the changes related to disabling sync on IO rgw daemons